src/main/resources/org/datadog/jmxfetch/new-gc-default-jmx-metrics.yaml

```diff
@@ -140,6 +140,52 @@
     domain: java.lang
     type: GarbageCollector
     name: ZGC Pauses
+    attribute:
+      CollectionCount:
+        alias: jvm.gc.zgc_pauses_collection_count
+        metric_type: counter
```

Comment:
Since these are new metrics, would it make sense to use `monotonic_count` type here?

Reply:
Sadly this isn't a new metric as reusing the old `jvm.gc.zgc_pauses_collection_count` that was created originally to support ZGC (added in #303).